grails-app/conf/Config.groovy

```diff
@@ -86,7 +86,7 @@ downloadAuth {
 }
 
 featureToggles {
-    // someFeature = <boolean>
+    dynamicGeoserverFilters = false
```

Comment:
Do we need a toggle?  I'd say no so long as all the geoservers are updated before the portal (which is likely).